background.js

```diff
@@ -257,27 +256,23 @@ async function linkResource(name, state) {
 }
 
 function handleFailure(payload) {
-  let errorCode, message, titlePart = null;
-  if (typeof (payload) == "number") {
+  let { errorCode, message, context: titlePart } = payload;
```

Comment:
Did eslint or Mark make this change?

Reply:
I did, in d2c994120ddbaf79289cd997ec8ded34348e3b87 (`ESLint: one-var: manual fixes`).